base.sbt

```diff
@@ -0,0 +1,2 @@
+addCommandAlias("gitSnapshots", ";set version in ThisBuild := git.gitDescribedVersion.value.get + \"-SNAPSHOT\"")
+enablePlugins(GitBranchPrompt)
```

Comment:
Ah I see,  you mean don't add it as standard, those that want it can always add it, but not vice-versa. Ok, will remove it